NEWS

```diff
@@ -1,3 +1,27 @@
+Release 3.5.5
+=============
+
+New features
+------------
+  
+- For ipython and Jupyter notebooks, R "magics" accepting optional input arguments
+  (`-i` or `--input`) can now be of form `<python-name>` or
+  `<r-name>=<python-name>`. The former is the way is has been working so far:
```

Comment:
Is it worth adding `<python-name>=<r-name>` to `-o` argument for symmetry as well?

Reply:
This could be added.

I am thinking that it would be better to bring checks that the destination exists to the beginning (before trying to evaluate the R cell). Having code run for potentially a long time and see it fail because a typo in the destination would be frustrating.

Reply:
Adding this without code duplication is proving trickier than I thought. I created issue #939 to track this in a future PR.